.github/workflows/common.yml

```diff
@@ -0,0 +1,38 @@
+name: Common CI workflow
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        ocaml-version:
+          - 5.1.0
+        node-version:
+          - hydrogen
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Cache node_modules
+        uses: actions/cache@v3
+        with:
+          path: jsland/node_modules
+          key: v1-${{ runner.os }}-${{ hashFiles('jsland/yarn.lock') }}
+
+      - name: Build and test
+        uses: kxcdev/ocaml-general-gha@v3
+        with:
+          ocaml-version: ${{ matrix.ocaml-version }}
+          node-version: ${{ matrix.node-version }}
+          setup-command: "opam install --locked . -y --deps-only --with-test"
+          build-command:
```

Comment:
maybe it's good to add an comment to explain about [this situation](https://github.com/kxcdev/kxc-melange-react-native-template/pull/3#issuecomment-1822281236) and as a result we run yarn as the build command to kxcdev/ocaml-general-gha

Reply:
I have added it.